Tests/PayID/PayIDIntegrationTests.swift

```diff
@@ -0,0 +1,79 @@
+import XCTest
+import XpringKit
+
+extension PaymentPointer {
+  /// The PayID to resolve.
+  public static let testPointer = "$dev.payid.xpring.money/hbergren"
```

Comment:
I assume we change payment pointer / PayID (what do you even call this thing now)? formats later?

Reply:
Or I guess this is blocked on the `payid` repo switching formats, correct?

Reply:
Now that the repo is updated, this can be switched to the new format.

Reply:
Updated